packages/components/src/unit-control/index.tsx

```diff
@@ -172,6 +172,7 @@ function UnforwardedUnitControl(
 			// matches the first character of a unit.
 			if (
 				! event.metaKey &&
+				! event.ctrlKey &&
```

Comment:
Let's also update the comment above (to reference the `ctrl` key in addition to the `meta` key).

Reply:
Thanks for the review! Updated by b90cc5a